libcxx/include/string

```diff
@@ -3939,9 +3939,22 @@ _LIBCPP_EXPORTED_FROM_ABI string to_string(long __val);
 _LIBCPP_EXPORTED_FROM_ABI string to_string(unsigned long __val);
 _LIBCPP_EXPORTED_FROM_ABI string to_string(long long __val);
 _LIBCPP_EXPORTED_FROM_ABI string to_string(unsigned long long __val);
+
+#  if _LIBCPP_STD_VER >= 26
+inline namespace __cpp26 {
+_LIBCPP_EXPORTED_FROM_ABI string __to_string(float __val);
+_LIBCPP_EXPORTED_FROM_ABI string __to_string(double __val);
+_LIBCPP_EXPORTED_FROM_ABI string __to_string(long double __val);
+
+inline _LIBCPP_HIDE_FROM_ABI string to_string(float __val) { return std::__to_string(__val); }
```

Comment:
I think we can use `to_chars` here to avoid changing ABI.

Reply:
See also my branch: https://github.com/frederick-vs-ja/llvm-project/tree/p2587r3. Do you think my approach viable?

Reply:
https://github.com/frederick-vs-ja/llvm-project/commit/78f270aeecece7a094578f1a52a10ae87904c627 - this looks like a better alternative to my experiments in implementing this.

BTW I wanted to get the CI green with what I have so far. I still haven't.

Would you like to update this PR with your changes or wait for whenever we can/decide to pursue this paper further or create a new PR?

Reply:
Since it seems it may take quite a while before `to_chars` will be done I don't think we need to spend time now to fix the CI, this will be outdated once we can pursue this patch.

That is unless somebody else volunteers to implement `to_chars` long double. It is on my TODO list, but not at a high priority. Realistically I don't expect to work on it this year.